cmd/helm/install.go

```diff
@@ -142,6 +142,7 @@ func addInstallFlags(f *pflag.FlagSet, client *action.Install, valueOpts *values
 	f.BoolVar(&client.Atomic, "atomic", false, "if set, installation process purges chart on fail. The --wait flag will be set automatically if --atomic is used")
 	f.BoolVar(&client.SkipCRDs, "skip-crds", false, "if set, no CRDs will be installed. By default, CRDs are installed if not already present")
 	f.BoolVar(&client.SubNotes, "render-subchart-notes", false, "if set, render subchart notes along with the parent")
+	f.StringVarP(&client.ReleaseName, "name", "", "", "The release name (and omit NAME parameter)")
```

Comment:
can you lowercase the capital T at the beginning of the flag description? That way it's consistent with the rest of the flag descriptions.

Reply:
Yes, good catch.